modules/run-vault/run-vault

```diff
@@ -366,7 +412,20 @@ function run {
   if [[ "$skip_vault_config" == "true" ]]; then
     log_info "The --skip-vault-config flag is set, so will not generate a default Vault config file."
   else
```

Comment:
Don't forget to add a check to enforce the requirement that if `--enable-auto-unseal` is true, then ` --auto-unseal-region`, ` --auto-unseal-kms-key`, and ` --auto-unseal-kms-key-region` are also required. For an example, see https://github.com/gruntwork-io/package-kafka/blob/master/modules/run-kafka/bin/run-kafka#L676-L687.